regression-test/suites/update/test_update_mow.groovy

```diff
@@ -16,10 +16,10 @@
 // under the License.
 
 suite("test_update_mow", "p0") {
-    def tbName1 = "test_update_unique_1"
-    def tbName2 = "test_update_unique_2"
-    def tbName3 = "test_update_unique_3"
-    def tbName4 = "test_update_unique_4"
+    def tbName1 = "test_update_mow_1"
```

Comment:
Has the ‘test_update_unique_2’ been used in another test case？

Reply:
yes. this testcase was copied from such a case originally. I think the writer forgets to modify it after coping.